exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -107,6 +108,8 @@ func (prwe *PrwExporter) PushMetrics(ctx context.Context, md pdata.Metrics) erro
 			if resourceMetric == nil {
 				continue
 			}
+
+			resource := resourceMetric.Resource
 			// TODO: add resource attributes as labels, probably in next PR
```

Comment:
should this TODO be marked fixed, or is there more to do?

Reply:
Let's keep it, we still need to do work to include resource attributes other than "instance" and "job".